stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -309,6 +313,9 @@ if(SWIFT_SHOULD_BUILD_EMBEDDED_STDLIB AND SWIFT_SHOULD_BUILD_EMBEDDED_CONCURRENC
       ${SWIFT_RUNTIME_CONCURRENCY_SWIFT_SOURCES}
       ${SWIFT_RUNTIME_CONCURRENCY_EMBEDDED_SWIFT_SOURCES}
 
+      GYB_SOURCES
+        ${SWIFT_CONCURRENCY_GYB_SOURCES}
```

Comment:
We need to do this and I've followed the pattern that e.g. the Synchronization module does but I'm not sure why I had the wrong SIA_ARCH in the other file when we try to do it here... Maybe it'd be better to just fix or workaround here rather than in the SetIfArchBitness? I wasn't quite sure how though